Detectors/ZDC/raw/include/ZDCRaw/DumpRaw.h

```diff
@@ -39,13 +40,17 @@ class DumpRaw
  private:
   void setStat(TH1* h);
   int mVerbosity = 1;
+  TH2* mTransmitted = nullptr;
```

Comment:
Hi @cortesep 
Please, either use unique_ptr for pointers (preferred) or add a destructor where you cleanup the histos

Reply:
Hi @shahor02,
Introduced unique_ptr for the histograms